src/platform/silabs/efr32/wifi/ethernetif.cpp

```diff
@@ -99,6 +115,7 @@ static void low_level_init(struct netif * netif)
 
     /* Accept broadcast address and ARP traffic */
     netif->flags |= NETIF_FLAG_BROADCAST | NETIF_FLAG_ETHARP | NETIF_FLAG_IGMP;
+    SILABS_LOG("%s: end", __func__);
```

Comment:
remove __func__  and use function name directly

Reply:
Done